src/content/concepts/mode.md

```diff
@@ -88,7 +88,7 @@ module.exports = {
 }
 ```
 
-If you want to change the behavior according the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
+If you want to change the behavior according to the **mode** variable inside the *webpack.config.js* you have to export a function instead of an object:
```

Comment:
The change is fine! I would also change`variable` to `property`, because it's not a variable.

Reply:
yes agreed